Charades_Game_In_JavaScript/Charades Game In JavaScript final blog.md

```diff
@@ -0,0 +1,348 @@
+    background: #f8e5a6;
+  }
+```
+
+**A friendly tip**: personally, I use [Visual Studio Code](https://code.visualstudio.com/) to write all my codes, since it has colors and makes it easier to read. You could try download it if you don't know where to start!
+
+Now that we have all our files created, we could move on to the first step -- creating the starting and ending page!
+
+![](https://i.imgur.com/9VQL74c.jpg)
+
+
+
+## Starting Page and Ending Page
+### Ending Page
+In the [live demo](https://), we could see that the ending page is just a page with one single line of words to show a finishing message.
+
+![](https://i.imgur.com/bBJIeDc.jpg)
+
+We can simply write in the HTML file.
+A simple header could be able to show the message, but we want it to be wrapped by a section to make it possible to center the text.
+
+```
+<body>
+    <section>
+        <h1>Times up!</h1>
+    </section>
+</body>
+```
+
+Since this is a very simple page, we don't need a script file to add function to it, and instead of having a separate CSS file to style the contents, we can simply add `style="..."` in each tag to customize the style.
+
+[](https://github.com/bitprj/Charades_Game_In_JavaScript/blob/master/end_code/endpage.html)
+
+
+### Starting Page
+![](https://i.imgur.com/2hL51bx.png)
+
+In the starting page, we would want to have a button so that the player could click on it and start the game. To create a button, use the `<button></button>` tag in HTML.
+
+You could play around with the design of the button in the corresponding **CSS** file "styles.css", and now let's focus on adding the effect when the button is clicked.
+
+##### **in HTML:**
+First, the tag `<button>` can have an **"onclick"** event, which would call the function `start()`
+![](https://i.imgur.com/aFkiBAS.png)
+
+To make the button work, we need to write the function in JavaScript.
+
+##### **in JavaScript:**
+We would want to go to the next page to choose the version when the start button is clicked.
+
+Our `function start() {}` would only have one line to jump to the next HTML page using `window.location`.
+
+![](https://i.imgur.com/eV0yayP.png)
+
+
+##### **in CSS:**
+Here is an example design of the start button, but you can absolutely customize yours by playing around with the features.
+
+![](https://i.imgur.com/hpqyiQl.png)
+
+
+
+## Tip: Linking Two HTML Files
+As we jump from the starting page to the next page, we change to another HTML file. In order to make this happen, we need to add another line of code in our HTML file.
+
+Use tag `<a href="{{YOUR_NEXT_HTML_NAME}}"></a>` under the `<head>` tag in your starting page's HTML, and now check out the page -- the start button should be working!
+
+![](https://i.imgur.com/HSesBXq.png)
+
+
+
+## Choosing the Game's Rule
+
+![](https://i.imgur.com/4yewZS5.png)
+
+The start button redirects us to the second page where we can choose the version we would like to play. One is limited time, the other is limited words.
```

Comment:
the image labels the two games differently than you did here. maybe make these explanations a little clearer